.infrastructure/i18n/extract-server-msgs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This needs to be run from beavy root like so:
+#     `./.infrastructure/i18n/extract-server-messages.sh`
+# Otherwise the working directory will be wrong.
+
+# Keys indicating the fn symbols that pybabel should search for
+# when finding translations.
+keys="-k format -k format_time -k format_date -k format_datetime"
```

Comment:
I just did for readability -- because it would get long. actually, i just realized i need to add the other functions to it. 

you would suggest placing it in the command directly?